app.js

```diff
@@ -3,27 +3,108 @@ const express    = require("express"),
 	  bodyParser = require("body-parser"),
 	  mongoose   = require("mongoose");
 	
+mongoose.connect("mongodb://localhost:27017/code_review", {useNewUrlParser: true, useUnifiedTopology: true});
 app.use(bodyParser.urlencoded({extended:true})); 
 app.set("view engine", "ejs");	
 
 // Serve static files
 var path = require('path');
 app.use(express.static(path.join(__dirname, 'static')));			  
-		  
-// Route for main page
+
+// 		  Mongo Schema for new reviews, eventually break off into seperate folder to req
+const reviewSchema = new mongoose.Schema({
```

Comment:
Do we need an ID field? for the detailed course page? How do we select the course to view?